lib/histogram.js

```diff
@@ -279,9 +279,16 @@ function convertLabelsAndValues(labels, value) {
 
 function extractBucketValuesForExport(histogram) {
 	return bucketData => {
-		const buckets = histogram.upperBounds.map(
-			createBucketValues(bucketData, histogram)
-		);
+		const buckets = [];
+		let acc = 0;
```

Comment:
I inlined this to avoid creating a function on a big loop, also made performance profiling cleaner. If you don't like it I can revert, but I did get a performance gain by inlining the method.

Reply:
I'm fine with this, especially if we now have benchmarks where it shows improvement